spec/lib/task-data/schemas/install-coreos-spec.js

```diff
@@ -16,7 +16,8 @@ describe(require('path').basename(__filename), function() {
         "installDisk": "/dev/sda",
         "comport": "ttyS0",
         "osType": "linux",
-        "rootPassword": "RackHD"
+        "rootPassword": "RackHD",
+        "ignitionScriptUri": "http://172.31.128.9090/api/current/templates/ignition.json"
```

Comment:
It's a minor suggestion for me, either updated it or not is OK. I have already approved your pull request. 
I will find an appropriate time to merge your pull request.

Thanks for your effort again.